parser/lexer.go

```diff
@@ -125,7 +125,7 @@ func (s *Scanner) skipWhitespace() rune {
 
 func (s *Scanner) scan() (tok int, pos Pos, lit string) {
 	ch0 := s.r.peek()
-	if isWhitespace(ch0) {
+	if unicode.IsSpace(ch0) {
```

Comment:
https://dev.mysql.com/doc/refman/5.7/en/identifiers.html

unquoted identifiers can't contain \r \n
if they appears in other place, quoted identifiers begin with ` , and strings begin with ' or ", all have special opening character.

I'll add some tests tomorrow.